src/store.ts

```diff
@@ -0,0 +1,34 @@
+import Vue from "vue";
+import Vuex from "vuex";
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    flags: 0,
+    bombsAmount: 10,
+    timeElapsed: 0
```

Comment:
w storze mielismy przechowywac caly stan gry. Chodzi o to żeby dodać opcję persisted do stora dzięki, której po odswieżeniu przegladarki zaladuje sie nam caly stan gry. W tym przypadku nie ma takiej opcji. Uzyj biblioteki https://github.com/gertqin/vuex-class-modules do stora.